### ISSUE TITLE
Have single handle gizmo not changing the transform of a node (or offset for ReflectionProbes)

### ISSUE THREAD

**Issue by jcostello** [godotengine/godot-proposals#7889]
### Describe the project you are working on

3D Lighting

### Describe the problem or limitation you are having in your project

The new single handled gizmo for resizing is great but for ReflectionProbes, moving each single gizmo changes the transform of the node and the offset of ReflectionProbes as show bellow. With alt you can evenly change the size of the Probes without changing the transform but not with single handled, which is not decired in this case por Probes because most of the time you don't want to modify the transform or offset when resizing

![Peek 2023-09-27 09-50](https://github.com/godotengine/godot-proposals/assets/1168582/04673901-41d8-4f38-8cf2-70c983b215ec)

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

It will be ideal for single handled gizmo modification to not change the transform or offset of a node

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

N/A

### If this enhancement will not be used often, can it be worked around with a few lines of script?

You have to recenter the node manually

### Is there a reason why this should be core and not an add-on in the asset library?

N/A

**Comment by Calinou** [comment]
What you're asking for is physically impossible, as reflection probes are always centered around their origin.

**Comment by Zireael07** [comment]
> as reflection probes are always centered around their origin.

That's not made clear enough in the editor itself then

**Comment by jcostello** [comment]
Maybe in this case, recalculate the offset to be in the same place an before, regarthing the center being in a new point

**Comment by jcostello** [comment]
Yeah, but the offset is where you want the probes to bake. Probably you want to move that even if the origin change
